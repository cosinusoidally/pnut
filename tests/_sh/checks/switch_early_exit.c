// expect_comp_failure
void main() {
  switch (1) {
    case 1:
      if (0) {
        break;
      } else {
        // This should not be allowed
      }
    case 2:
      return;
  }
}
