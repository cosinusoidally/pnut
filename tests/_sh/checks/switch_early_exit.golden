tests/_sh/checks/switch_early_exit.c:14:0  Early break out of a switch case is unsupported
